### PR TITLE
added andes meta.yaml

### PR DESCRIPTION
**Pull request by cuihantao**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by cuihantao** [comment]
Posting a comment willing to be listed.

**Comment by cuihantao** [comment]
@conda-forge/staged-recipes

I would like to ping and let you know that the recipe has passed all checks.

**Comment by cuihantao** [comment]
@conda-forge/staged-recipes

Would like to get some attention for action but no hurries. Thanks.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +1,201 @@
-Copyright (c) 2015-2018, conda-forge
```
Comment (mariusvniekerk) [inline LICENSE.txt]: place your changed license file in the recipe dir itself

File: recipes/andes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - andesplot --help
+
+about:
+  home: https://github.com/cuihantao/andes
+  license: Apache Software
+  license_family: APACHE
+  license_file: 
+  summary: Another Power System Simulator
+  doc_url: https://andes.readthedocs.io 
+  dev_url:
```
Comment (mariusvniekerk) [inline recipes/andes/meta.yaml]: Set the dev_url as the github repo

File: recipes/andes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - andes.variables
+  commands:
+    - andes --help
+    - andesplot --help
+
+about:
+  home: https://github.com/cuihantao/andes
+  license: Apache Software
+  license_family: APACHE
+  license_file:
```
Comment (mariusvniekerk) [inline recipes/andes/meta.yaml]: You need to mention the license file by name

File: recipes/andes/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+requirements:
+  host:
+    - cvxopt
+    - matplotlib
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
```
Comment (mariusvniekerk) [inline recipes/andes/meta.yaml]: This probably just needs to be python and pip since this looks like its a pure python package

**Comment by cuihantao** [comment]
I have made necessary changes to resolve the requested changed. Thank you! @mariusvniekerk

**Pull request merged**